packages/fuels/Cargo.toml

```diff
@@ -45,3 +45,4 @@ std = [
 # TODO: To be removed once https://github.com/FuelLabs/fuels-rs/issues/881 is unblocked.
 test-type-paths = []
 fuel-core-lib = ["fuels-test-helpers?/fuel-core-lib", "dep:fuel-core"]
+rocksdb = ["fuels-test-helpers?/rocksdb"]
```

Comment:
This is a bit convoluted. This line means the following:
If `fuel-test-helpers` is enabled, then enable its `rocksdb` feature. 

If we take a look at the `rocksdb` feature definition over in `fuel-test-helpers` it says:
```toml
rocksdb = ["fuel-core", "fuel-core/rocksdb"]
```

So this creates the following situation:
* If you enable `fuel-core-lib` and `rocksdb` features but don't enable `fuels-test-helpers` you're not going to get the `rocksdb` feature enabled on `fuel-core` because `fuels-test-helpers` was in charge of that.

The fix depends on what you decide to do with regard to feature flags in one of my previous comments.